### ISSUE TITLE
Increasing the pop up definition text size or maybe add the possibility to increase it manually

### ISSUE THREAD

**Issue by emanuelps2708** [jzohrab/lute-v3#212]
**Is your feature request related to a problem? Please describe.**

Hi jz, maybe this is specific to my computer only but the text on the pop up is a bit small so it's difficult for me to read the text that I have added.

**Describe the solution you'd like**

Increasing the default pop up text size a little bit or maybe add the possibility to adjust it manually

![image](https://github.com/jzohrab/lute-v3/assets/147004033/ea2336f4-906f-4e49-a8ed-aa302e20d7ad)

**Comment by jzohrab** [comment]
Hi @emanuelps2708 , thanks for the note.

You could try a custom style.  For example, here's the default hover:

> <img width="111" alt="image" src="https://github.com/jzohrab/lute-v3/assets/1637133/b1bf9ce9-be4d-4a05-9fa9-892c508ff18f">

If I add the custom style `div.ui-tooltip {  font-size: 150%; }` to the settings (home page - settings - settings), it looks like this:

> <img width="159" alt="image" src="https://github.com/jzohrab/lute-v3/assets/1637133/68e17474-8ce7-4f18-852d-d2ff67c6c587">

Give that a shot, and if it works for you, I'll add a note to the styling page in the manual.  (It's hard to know how to change things unless you already know CSS etc -- maybe some day that will change, but for now this hacky approach works).

Thanks!

**Comment by emanuelps2708** [comment]
It works perfectly, is just what I needed, thank you so much for helping me ;)

![image](https://github.com/jzohrab/lute-v3/assets/147004033/120a223e-15d3-415d-8b1d-cf1fd6b06329)

**Comment by M-Biggles** [comment]
Nice hack, @jzohrab!

I'll start going blind in my 40s, so this will be helpful. I'll increase 5% every few years! :D

**Comment by jzohrab** [comment]
Yeah it was a bit tiny, wasn't it!  Super that this works.  I've added it to the manual, https://jzohrab.github.io/lute-manual/usage/themes/custom-styles.html.  It will be there in a few minutes.  Cheers and thx for the issue!
